contrib/slime/README.org

```diff
@@ -74,6 +74,15 @@ of Common Lisp, you can specify it in your =~/.spacemacs=
 | ~SPC m c f~ | Compile function         |
 | ~SPC m c r~ | Compile region           |
 
+** Definition
+
+| Key Binding               | Description                     |
+|---------------------------+---------------------------------|
+| ~SPC m d d~               | Go to definition                |
+| ~SPC m d u~ or ~SPC m d D | Return from definition          |
```

Comment:
I find the ` character a little awkward to type, myself. Other possibles:
SPC m g b(ack), SPC m g , (in reference to the emacs/slime M-. M-, binding
pair, but then for consistency the SPC m g g which you recommend above
should have SPC m g . as an alternative.)

On Thu, Aug 13, 2015 at 7:33 PM, Sylvain Benner <notifications@github.com>
wrote:

> In contrib/slime/README.org
> <https://github.com/syl20bnr/spacemacs/pull/2648#discussion_r37045617>:
>
> > @@ -74,6 +74,15 @@ of Common Lisp, you can specify it in your =~/.spacemacs=
> >  | ~SPC m c f~ | Compile function         |
> >  | ~SPC m c r~ | Compile region           |
> >
> > +** Definition
> > +
> > +| Key Binding               | Description                     |
> > +|---------------------------+---------------------------------|
> > +| ~SPC m d d~               | Go to definition                |
> > +| ~SPC m d u~ or ~SPC m d D | Return from definition          |
>
> There is no conventions for this one but I would like to put this
> pop-stack on SPC m g ` which matches SPC ` and ``
>
> SPC m g u is already taken for go to usages.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/pull/2648/files#r37045617>.
>